### PR TITLE
DPEL site migration v1

### PR DESCRIPTION
**Pull request by cecilia-dwa**
Please check if the files are okay and check the READ.ME files especially to make sure all instructions and info need to be there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,77 @@
...
+5.) Copy and paste the markdown layout from `alab-license.md`. Fill/Edit in the appropriate info as needed.
+6.) Go back to the `index.astro` file under the project's license folder you set up earlier from /pages. Update this line with your markdown (`.md`) file name --> `const licenseContent = await Astro.glob("../../../content/license/license-markdown-file-name.md");`
+7.) Save your files and run `npm start dev` to run the project.
+8.) You can check this page by going to the name of the folder you set under the pages/ folder.
+
+## There are images on the page
+All images should be stored in the public/images/ folder. It is important that they are separated by project/page topic that they will be under for organization. 
+
+## Still need help? 
+Please reach out to cecilia.tran@dreamworks.com or matthew.low@dreamworks.com if any questions.
```
Comment (matthewlow-dwa) [inline README.md]: Let's change this to:
> Please reach out to `assets-discussion@lists.aswf.io` or find us on the [ASWF Slack](https://slack.aswf.io/) at `#assets`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,77 @@
...
+A library of digital assets - 3D scenes, digital cinema footage, etc. - that demonstrate the scale and complexity of modern feature film production, including computer graphics, visual effects and animation.
+
+## 🚀 Project Structure
+
+Inside the DPEL project, you'll see the following folders and files:
+
+```
+/
+├── public/
+│   └── favicon.svg ( This is the icon that shows up in the browser tab )
```
Comment (matthewlow-dwa) [inline README.md:13]: It looks like this file is currently the Astro icon.  Should this be updated with the ASWF DPEL icon?

File: README.md (hunk 1 above)
Comment (cecilia-dwa) [reply README.md]: Updated :)

File: README.md (hunk 2 above)
Comment (cecilia-dwa) [reply README.md:13]: I'll update that!

File: public/favicon.svg
Comment (cecilia-dwa) [inline public/favicon.svg:1]: updated favicon to the DPEL icon

File: README.md (hunk 2 above)
Comment (cecilia-dwa) [reply README.md:13]: Okay it's been updated, please check again to make sure!

**Review by matthewlow-dwa — APPROVED** [review]
LGTM!

**Pull request merged**